dev-packages/application-manager/src/generator/frontend-generator.ts

```diff
@@ -53,7 +53,7 @@ export class FrontendGenerator extends AbstractGenerator {
 </head>
 
 <body>
-  <div class="theia-preload">${this.compileIndexPreload(frontendModules)}</div>
+  <div class="theia-preload theia-loader">${this.compileIndexPreload(frontendModules)}</div>
```

Comment:
I believe the following approach breaks downstream applications which use `theia-preload` to define their own custom spinners. At the moment, they may provide their own loading animation or splash screen with `compileIndexPreload` and the `theia-loader` styling would need to be overwritten.

Reply:
I have no idea how do we make a css style just disappear instead of overriding it when somebody defines `.theia-preload`

Reply:
I don’t think anyone is asking for stylings to disappear, only that the new classname and introduction of the codicon-refresh spinner break downstream applications unnecessarily (ex: who have introduced their own preload template). 

The following may be more aligned with what the changes should be: https://github.com/eclipse-theia/theia/pull/7513#discussion_r406039856